### PR TITLE
Only include paired-end sequencing reads in discordants file

### PR DESCRIPTION
**Pull request by ernfrid**
I've encountered a few CRAMs in the wild where there are reads that aren't from paired-end sequencing runs. These did not have the BAM_FMUNMAP bit set so I've added a check that the sequencing run was paired to the section writing to the discordants file.

I also reindented to remove a bunch of tabs.

I've a test in process and the code change seems to help, but please review critically. cc @brentp @ryanlayer 

Thanks!

### CODE REVIEW

File: src/filter/filter.c [hunk 1]
```diff
@@ -266,49 +266,49 @@ int main(int argc, char **argv)
...
     while(ret = sam_read1(in, hdr, aln) >= 0) {
         if (((aln->core.flag) & (BAM_FUNMAP | BAM_FQCFAIL | BAM_FDUP)) != 0)
             continue;
 
-		dropped = 0;
-        if (((aln->core.flag) & (BAM_FPROPER_PAIR | BAM_FMUNMAP | BAM_FSUPPLEMENTARY)) == 0) {
-			dropped = 1;
-			aln = drop_seq_qual(aln);
+        dropped = 0;
+        if ((((aln->core.flag) & (BAM_FPROPER_PAIR | BAM_FMUNMAP | BAM_FSUPPLEMENTARY)) == 0) && (aln->core.flag & BAM_FPAIRED == 1)) {
```
Comment (brentp) [inline src/filter/filter.c]: this is the only line that changed, yes?

Comment (ernfrid) [reply src/filter/filter.c]: That's correct. This view may prove easier to review: https://github.com/arq5x/lumpy-sv/pull/279/files?utf8=%E2%9C%93&diff=split&w=1

Comment (brentp) [reply src/filter/filter.c]: looks good to me.

**Comment by ernfrid** [comment]
If this looks good, is it possible to merge so I can build against the official repo instead of my fork? Thanks!

**Comment by brentp** [comment]
CC @ryanlayer @arq5x

**Pull request merged**